src/features/builder/Builder.tsx

```diff
@@ -0,0 +1,121 @@
+import {
+  Box,
+  Flex,
+  Tab,
+  TabList,
+  TabPanel,
+  TabPanels,
+  Tabs,
+} from '@chakra-ui/react';
+import { Component, NodeType } from '@metamask/snaps-ui';
+import { NodeModel } from '@minoru/react-dnd-treeview';
+import { FunctionComponent, useState } from 'react';
+
+import { Editor } from '../../components';
+import { TemplateComponentList, NodeTree, NodeRenderer } from './components';
+import { nodeModelsToComponent, panelToCode } from './utils';
+
+export const Builder: FunctionComponent = () => {
+  const [id, setId] = useState<number>(100);
```

Comment:
Why are we starting at `100`?

Reply:
It was done like this in the example. It makes sure there's no overlap between the default nodes and the newly created nodes. In this case we can just start at 2 though.